### ISSUE TITLE
I can not import any json. the button is not working after clicking it. 

### ISSUE THREAD

**Issue by needhelp256** [airbnb/lottie-web#2972]
**Tell us about your environment**

* **Browser and Browser Version:**
* **After Effects Version:**

**What did you do? Please explain the steps you took before you encountered the problem.**

**What did you expect to happen?**

**What actually happened? Please include as much _relevant_ detail as possible.**

**Please provide a download link to the After Effects file that demonstrates the problem.**

**Comment by Djsquid2020** [comment]
+1 Unable to import local JSON file. Running After effects v23.3.0 (Build 53) and Bodymovin v5.11.0

**Comment by fergusmeiklejohn** [comment]
Yeah I raised this issue a couple of weeks ago: https://github.com/airbnb/lottie-web/issues/2963#issue-1662151020

**Comment by edwardluong** [comment]
+1 Unable to import local JSON file. Running After effects v23.1.0 (Build 83) and Bodymovin v5.12.0

Issue: I imported json file from local+URL -> Bodymovin said import finished -> But no file imported to my AE Projects
